core/handler/tracing_handler.go

```diff
@@ -236,6 +230,22 @@ func (t *TracingConsumerHandler) Handle(chain *Chain, i *invocation.Invocation,
 	span.Finish()
 }
 
+func checkProtocolAndSetinterfaceName(interfaceName string, i *invocation.Invocation) string {
```

Comment:
Suggestion :  Change Name to setInterfaceName(interfaceName string, i *invocation.Invocation)